packages/dev-server-hmr/test/browser.test.ts

```diff
@@ -33,7 +41,8 @@ describe('browser tests', function () {
     await browser.close();
   });
 
-  it('should bubble when bubbles is true', async () => {
+  it('should bubble when bubbles is true', async function () {
+    this.timeout(3000);
```

Comment:
For my local machine the test ran in just over two seconds. If CI takes longer then it will be represented in the CI runs of this PR and we can tweak it then when we know how long it takes on CI.

Reply:
Sorry I'm on mobile. Looks like CI finished already and ran green indicating that the test isn't running slow due to CPU constraints.

Reply:
I meant the test might become flaky
like when you need to restart tests often because they fail with timeout errors cause by a stricter time constraints
but I'm OK to tweak or revert this specific part if it starts happening